### DISCUSSION TITLE
How do I install this thing?

### DISCUSSION THREAD

**Discussion by VladimirLenin1917** [younesaassila/ttv-lol-pro#159] [General]
Hey just made an account to ask this.

How do you actually install this? I'm a bit of a noob and the instructions on the code page (main page for the addon w/e) arn't quite clear?

Which file do you install when you get to the "Install Add-on From File" step? None of the files that are extracted from the Firefox zip are XPI files?

If anyone would assist, would appreciate it :)

**Comment by rafito7** [comment]
if you are talking about V2, it works on firefox dev (https://www.mozilla.org/en-US/firefox/developer/), there you install the zip without unzipping (https://github.com/younesaassila/ttv-lol-pro/releases/download/v2.0.0-rc/FIREFOX-ttvlolpro-v2.0.0.zip)
